src/main/java/org/tron/core/services/WitnessService.java

```diff
@@ -116,6 +121,22 @@ public WitnessService(Application tronApp, AnnotationConfigApplicationContext co
         }
       };
 
+
+  /**
+   * Cycle thread to repush Transactions
+   */
+  private Runnable repushLoop =
+      () -> {
+        while (isRunning) {
+          try {
+            TransactionCapsule tx = this.tronApp.getDbManager().getRepushTransactions().take();
+            this.tronApp.getDbManager().rePush(tx);
+          } catch (InterruptedException e) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Either re-interrupt this method or rethrow the "InterruptedException". ([squid:S2142](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2142))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1298&issues=AWUzb7VUW2AOP0CkArPp&open=AWUzb7VUW2AOP0CkArPp)

[ISSUE_KEY]: AWUzb7VUW2AOP0CkArPp